Makefile

```diff
@@ -44,8 +44,10 @@ PKG := ${PWD:${GOPATH}/%=%}
 update-codegen-crds:
 	docker run -v ${PWD}:/go/$(PKG):Z -w /go/$(PKG) registry.svc.ci.openshift.org/ocp/4.2:crd-schema-gen --apis-dir vendor/github.com/openshift/api/operator/v1
 update-codegen: update-codegen-crds
+
+TMP_GOPATH :=$(shell mktemp -d)
 verify-codegen-crds:
-	go get github.com/openshift/crd-schema-gen/... && go install github.com/openshift/crd-schema-gen/...
+	export GOPATH=$(TMP_GOPATH) && export GOBIN=$(TMP_GOPATH)/bin && git clone -b release-4.2 --single-branch --depth 1 https://github.com/openshift/crd-schema-gen.git $(TMP_GOPATH)/src/github.com/openshift/crd-schema-gen && go install $(TMP_GOPATH)/src/github.com/openshift/crd-schema-gen/cmd/crd-schema-gen
```

Comment:
`GOPATH=$(TMP_GOPATH) GOBIN=... go install ...` is enough. No need to export. git does not use them.

Reply:
sure about GOBIN? We have to call it from there as well.

Reply:
some people like @stlaz have set GOBIN explicitly and without setting it to temp we install it to the global GOBIN instead the one in temporary GOPATH